src/roi/Roi.ts

```diff
@@ -40,8 +73,12 @@ export class Roi {
     this.width = 0;
     this.height = 0;
     this.surface = 0;
+    this.computed = {};
+    this.minX = Infinity;
+    this.minY = Infinity;
+    this.maxX = -Infinity;
+    this.maxY = -Infinity;
```

Comment:
I think those 4 parameters should disappear and are replaced by `origin` and `width`/ `height`